src/components/Table/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, memo } from "react";
+
+import Header from "./Header";
+import Rows from "./Rows";
+import "./Table.style.scss";
+import { IEvent, IProps } from "./Table.types";
+
+const Table = (props: IProps) => {
+  const [sort, setSort] = useState("");
+  const [reload, setReload] = useState(false);
+  const { data } = props;
+
+  function compareNumbers(a: any, b: any, columnId: string) {
+    const nameA = a[columnId];
+    const nameB = b[columnId];
+
+    if (!nameA || !nameB) return 0;
+
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+
+    return 0;
+  }
+
+  const handleSortColumn = (e: IEvent) => {
+    const columnId = e.target.id;
+    if (columnId === sort) {
+      data.reverse();
```

Comment:
Reverse() method mutate data table, you want to do it? If yes, forget about this comment :)

Reply:
Yeah, here I want to mutate data table. Do you think this is not a good practice/idea?

Reply:
If you want to mutate, there is no problem with that :)